projects/packages/my-jetpack/src/class-initializer.php

```diff
@@ -260,6 +261,7 @@ public static function enqueue_scripts() {
 				),
 				'latestBoostSpeedScores' => $latest_score,
 				'scanData'               => $scan_data,
+				'wafConfig'              => Waf_Runner::get_config(),
```

Comment:
I wonder if we could have the `scanData` and `wafConfig` under the same parent key? I know they are different but they are under the same plugin so I think it would look neater if they were under an umbrella key for all scan related data 🤔

Reply:
Yes, that would be good, will do. 👍